training/training/routes/datasets/default/columns.py

```diff
@@ -1,18 +1,25 @@
 from django.http import HttpRequest
 from ninja import Router, Schema
+from ninja.security import HttpBearer
 from training.core.dataset import SklearnDatasetCreator
 from training.routes.datasets.default.schemas import DefaultDatasetResponse
 from training.routes.schemas import NotFoundError
+from core.authenticator import FirebaseAuth
```

Comment:
import should be `training.core.authenticator`

Reply:
Fixed!